express/scripts/scripts.js

```diff
@@ -2225,14 +2225,41 @@ export async function addFreePlanWidget(elem) {
+    const bullet = createTag('div', { class: 'free-plan-bullet' });
     const widget = createTag('div', { class: 'free-plan-widget' });
+    const optoutButton = createTag('button', { class: 'free-plan-optout' });
+
     widget.innerHTML = `
       <div><div>${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
       <div><div>${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
     `;
+    bullet.innerHTML = `
+      <div class="free-plan-bullet-container">
+        <div class="free-plan-bullet-tray">
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #c457f0;">${checkmark}</div><div>${placeholders['free-plan-check-1']}</div></div>
+          <div class="free-plan-tag"><div style="background-color: #f06dad;">${checkmark}</div><div>${placeholders['free-plan-check-2']}</div></div>
+        </div>
+      </div>
+    `;
```

Comment:
This feels like a lot of repetitive that could be concatenated programmatically in 2-3 lines...